src/Symfony/Bundle/FrameworkBundle/EventListener/SuggestMissingPackageSubscriber.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\EventListener;
+
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleErrorEvent;
+use Symfony\Component\Console\Exception\NamespaceNotFoundException;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Suggests a package, that should be installed (via composer),
+ * if the package is missing, and the input command namespace can be mapped to a Symfony bundle.
+ *
+ * @author Przemysław Bogusz <przemyslaw.bogusz@tubotax.pl>
+ */
+final class SuggestMissingPackageSubscriber implements EventSubscriberInterface
+{
+    const SUGGESTED_PACKAGES = [
```

Comment:
Mark private?  I don't think there's a reason this needs to be publicly accessible.

Reply:
Changed into private static property, as it seemed to be in line with other Symfony's classes.